flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Engine.js

```diff
@@ -284,5 +233,63 @@ var A320_Neo_LowerECAM_Engine;
         }
     }
     A320_Neo_LowerECAM_Engine.EngineInfo = EngineInfo;
+    class EngineIgniter {
+        constructor(_ignValueText) {
+            this.currentState = A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.NONE;
```

Comment:
**thought**: `EngineIgniter` is located in the same scope as `Definitions`.
**suggestion**: Remove the `A320_Neo_LowerECAM_Engine` prefix here and in other locations. Shorter but same result 😀.

Reply:
Replaced all occurences of 
`A320_Neo_LowerECAM_Engine.Definitions.*`
with
`Definitions.*`